### PR TITLE
implement options to mecab plugin that extracts specific part-of-speech (fix #1073)

### LINKED ISSUE #1073: MeCab plugin: support extracting specific part-of-speech
Using specific type of part-of-speech (`品詞`) to construct feature vector is a common requirement.
For example, users may want to only use nouns to extract topics from the document.

Currently we users need to do that in client side, which is costful.  So I want to add that feature to MeCab plugin.

As words in MeCab dictionaries have hierarchical (4 layered) part-of-speech (e.g., `名詞/固有名詞/人名/姓`), we'll need to design some kind of simple query language in JSON, like this:

```
{
  "include": ["名詞/*", "動詞/*"],
  "exclude": ["名詞/固有名詞/*"]
}
```

### PR DESCRIPTION
**Pull request by kmaehashi**
Fix #1073.

This patch depends on https://github.com/jubatus/jubatus_core/pull/256

### CODE REVIEW

**Comment by rimms** [comment]
Great works! :+1: 

I tested the following conditions:

* Config:

   ```js
   {
     "converter": {
       "string_types": {
           "mecab": {
             "method": "dynamic",
             "path": "libmecab_splitter.so",
             "function": "create",
             "include_features": "名詞,*",
             "exclude_features": "名詞,形容動詞語幹,*"
           }
       },
       "string_rules": [
           { "key": "*", "type": "mecab", "sample_weight": "bin", "global_weight": "bin" }
       ]
     },
     "method": "PA"
   }
   ```

* Data:

   ```
   Jubatusに新たな機能が追加されます
   ```

* Expected feature vector

   ```
   "Jubatus": 1
   "機能": 1
   "追加": 1
   ```

* Actual feature vector (confirmed via ``jubadump``)

   ```
   name$追加@mecab#bin\/bin: 1
   name$機能@mecab#bin\/bin: 1
   name$Jubatus@mecab#bin\/bin: 1
   ```

File: plugin/src/fv_converter/mecab_splitter.cpp [hunk 1]
```diff
@@ -55,6 +59,29 @@ mecab_splitter::mecab_splitter(const char* arg, size_t ngram, bool base)
     throw JUBATUS_EXCEPTION(
         converter_exception("ngram must be a positive number"));
   }
+
+  if (include.empty()) {
+    throw JUBATUS_EXCEPTION(
+        converter_exception("include must not be empty"));
```
Comment (rimms) [inline plugin/src/fv_converter/mecab_splitter.cpp]: s/include/include_features/

**Comment by rimms** [comment]
:+1: LGTM other than the above comment.

**Comment by kmaehashi** [comment]
Fixed.

**Pull request merged**